src/components/Comments.jsx

```diff
@@ -1,17 +1,46 @@
 import { useEffect, useState } from "react";
-import { getComments } from "../../api";
+import { getComments, postComment } from "../../api";
 
 export const Comments = ({article_id}) => {
     const [comments, setComments] = useState([])
+    const [commentInput, setCommentInput] = useState("")
     
     useEffect(() => {
+        // if(newComment != ""){
+        //     postComment(article_id, newComment).then((data) => setComments(data, ...comments))
+        //     setNewComment("");        
+        // }
         getComments(article_id).then((data) => {
             setComments(data)
         })
-    },[article_id])
+    },[article_id, comments])
+
+    const handleChange = (e) => {
+        setCommentInput(e.target.value);
+      };
+    
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const newComment = commentInput;
+        setComments([newComment, ...comments]) 
+        setCommentInput("");
+        postComment(article_id, newComment).then((data) => setComments(data, ...comments))
+        console.log(comments)
+      };
 
     return (
         <section className="comment-section">
+            <form className="new-comment" onSubmit={handleSubmit}>
+                <label htmlFor="comment-input">
+                    Add Comment:
+                    <input
```

Comment:
I would recommend a text area over an input as a text area allows you more than one line of text.